packages/amplify/amplify_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 1.0.0
+
+We are thrilled to release version 1.0 of the Amplify Flutter libraries!
+
+This release introduces a full rewrite of the Amplify libraries in Dart, allowing you to build AWS-powered
+applications on every platform Flutter supports (iOS, Android, Web, macOS, Windows, and Linux).
+
+If you've been using v0 in production, we believe you can now safely migrate to v1. Check out our 
```

Comment:
nit/question: Should all these be changed to "in production, you can now ..." to remove "we believe." ?

Reply:
Done 👍